.vscode/tasks.json

```diff
@@ -73,12 +73,6 @@
         }
       }
     },
-    {
-      "label": "Create Release Branch",
-      "command": "./scripts/create-release-branch-workflow.sh",
-      "type": "shell",
-      "args": ["${input:circleCiId}", "${input:publishType}"]
-    },
```

Comment:
hum....have you seen any docs about trigging GH actions remotely?  I guess we now have the button in GHA so maybe not necessary.

Reply:
There are actually some VS Code extensions that will help you do it! I think it's supposed to be pretty easy once you connect and auth in, but we can iterate on that as we go.